docs/formats/zst.md

```diff
@@ -190,128 +117,132 @@ independently of each other, similar to zng compression framing.
 ### The Reassembly Section
 
 The reassembly section provides the information needed to reconstruct
-column streams from segments, and in turn, to reconstruct the original zng rows
-from column streams, i.e., to map columns back to rows.
+column streams from segments, and in turn, to reconstruct the original Zed values
+from column streams, i.e., to map columns back to composite values.
 
 > Of course, the reassembly section also provides the ability to extract just subsets of columns
 > to be read and searched efficiently without ever needing to reconstruct
 > the original rows.  How performant this is all done is up to any particular
-> zst implementation.
+> ZST implementation.
 
 > Also, the reassembly section is in generally vastly smaller than the data section
 > so the goal here isn't to express information in cute and obscure compact forms
 > but rather to represent data in easy-to-digest, programmer-friendly form that
-> leverages zng.
+> leverages ZNG.
 
-The reassembly section is a zng stream.  Unlike parquet,
+The reassembly section is a ZNG stream.  Unlike Parquet,
 which uses an externally described schema
 (via [Thrift](https://thrift.apache.org/)) to describe
-analogous data structures, we simply reuse zng here.
+analogous data structures, we simply reuse ZNG here.
 
```

Comment:
I found myself getting confused about what the reassembly section looks like from a 1000 foot view.  I would include a brief overview of what the reassembly section contains before diving into the details. Something like:
```suggestion
This reassembly stream encodes 2*N+1 Zed values, where N is equal to the number of top-level Zed types (referred to as [super types](#the-super-types)) that are present in the ZST Object. These Zed values are split into two sections:

|  section | # of values | descriptions |
| --- | --- |
| [Super Types](#the-super-types) | N | Describes the type of each top level Zed value. |
| [Reassembly Records](#the-reassembly-records) | N+1 | Maps the locations of column streams to each super type (+1 for the [super column|(#the-super-column) |

Reply:
This impacts the overall structure of the sections so I would like to defer making improvements here to a subsequent PR.